### ISSUE TITLE
Board Migration

### ISSUE THREAD

**Issue by edwardsarah** [hackforla/guides#168]
### Overview
We need to migrate the Guides: Product Management project board from classic to a new project board on GitHub

### Action Items
- [ ] Copy all content from project board cards into this issue to preserve it
- [ ] Any cards that describe a column are added to the column header
- [ ] Update wiki page with content from project board cards (likely under Joining the Team, but we will discuss depending on the content of each card)

### Resources/Instructions
- [Guides: Product Management Classic Board](https://github.com/hackforla/guides/projects/1)
- [Wiki: Joining the Team](https://github.com/hackforla/guides/wiki/Joining-the-Team)

**Comment by edwardsarah** [comment]
Project Board Card content & Where it Should Go

### Start Here

- Start Here = Includes things like meeting times and zoom links, and anything the team members need to read. Cards that have links that you need to get to often. 
  - **Likely obsolete, as we will not have cards on our new board**
- Meeting times:
Friday
7:30am PDT
8:00pm IST
3:30pm WAT
4:30pm CET
  - **Wiki & project readme (where it should be already)**
- Links
[Guides Project Readme](https://github.com/hackforla/guides/blob/main/README.md)
[Guides Team One-Sheet](https://github.com/hackforla/guides/issues/8)
[All Hack for LA Guides Shortcuts](https://drive.google.com/drive/folders/1buzmMJ5gCMvm3mpmdqDWryI6IBA8foKA)
[Organizations Google Drive: Hack for LA Guides](https://drive.google.com/drive/folders/1LAE_-KUd72u7vTBOTwU1Rz83rTCbeWB_?usp=sharing)
[Guides Team Figma](https://www.figma.com/file/Jz8KoGTBIxdx9jRxBWrEsF/Guides-Team-Figma?node-id=0%3A1)
[Communities of Practice](https://github.com/hackforla/communities-of-practice)
[All Guides Tracker Project board](https://github.com/orgs/hackforla/projects/3#card-72148101)
[Cleanup Guide Templates Issues](https://github.com/hackforla/guides/issues?q=is%3Aopen+is%3Aissue)
[Audit Spreadsheet for Tracking CoPs Guides](https://docs.google.com/spreadsheets/d/1mO3MPGPs3fD45XI1yuVeToqROS6n4rmY8ciGjVst9sc/edit#gid=0)
[Guides Team Slide Deck for Guide Creation](https://github.com/hackforla/guides/issues/44)
  - **Wiki: Joining the Team**

### Icebox
- Icebox = issues identified that are going to get addressed, but are not being started yet because of other more pressing issues.  Or issues that are not fully developed.
  - **Column Header**

### New Issue Approval
- Where new issues get started and become ready for moving into the prioritized backlog
  - **Column Header**
- Add new team members to Google Drive, Have them add themselves to the roster, Add them to Figma
- Bi-Weekly check on Guides that need leadership review
[Product Management](https://github.com/hackforla/product-management/issues?q=is%3Aopen+is%3Aissue+label%3A%22TG%3A+Leadership+Review%22)
[UI/UX](https://github.com/hackforla/UI-UX/issues?q=is%3Aopen+is%3Aissue+label%3A%22Guide%3A+Leadership+Review%22)
  - **Add to Agenda template?**
- Follow up with this guide author
https://github.com/hackforla/knowledgebase-content/issues/84
  - **Add to agenda to discuss**
- Make issue for emojis (they need to be useable across platforms) see emojiterra.com for compatability
  - **Create new issue - is this a guide?**

### Prioritized Backlog
- Prioritized Backlog = features that are ready to work and a priority.  This column should be prioritized by most important first.
  - **Column Header**

### In Progress
- In Progress = tasks that are actively being worked on by an individual team member and have been assigned to them in the issue.
  -  **Column Header**

### Questions / Review
need to add new text to issues under peer review section
https://github.com/hackforla/knowledgebase-content/issues/127
- [x] Review the guide with UX/UI community of practice
   - [ ] Make any changes indicated by comments
   - [ ] Ask for new reviews until no suggestions for improvements are left
   - [ ] Once done, remove the "Guide: Review Guide" label and add the "Guide: Leadership Review" label
   - **No longer necessary with new Guides template**
- Questions/Review =  Issues that you are blocked on because you need an answer from another team member or other type of question/review.
  -  **Column Header**

### To Be Addressed
- Check the Wiki to Update It
- Aeding indicated on Slack that she would like to rejoin the team. https://hackforla.slack.com/archives/C028T9XU9S5/p1659704557772719
  - **likely no longer relevant, but can make an issue or agenda item to follow up**
- We need to define some other Onboarding and Offboarding situations
  - **New issue, although it's unclear what Bukki meant by this**
- We need to create an issue or a guide that explains all the labels in Hack For LA and how/when to use them
  - **New issue, knowledgebase-content**
- We need to add Guide for making guides issue/templates, draft, and the guide for adding screenshots to the Resource Section of new guide template across all CoPs. See Product Management Guides for reference

**Comment by edwardsarah** [comment]
### Text from project board card, Start Here
```
Start Here = Includes things like meeting times and zoom links, and anything the team members need to read. Cards that have links that you need to get to often. 
```
- [ ] Determine where this item goes on the wiki (page and section)
- [ ] copy item to wiki
- [ ] hide this comment when completed.

**Comment by edwardsarah** [comment]
### Text from project board card, Start here
```
Meeting times:
Friday
7:30am PDT
8:00pm IST
3:30pm WAT
4:30pm CET
```
- [ ] Determine where this item goes on the wiki (page and section)
- [ ] copy item to wiki
- [ ] hide this comment when completed.

**Comment by edwardsarah** [comment]
### Text from project board card, Start Here
```
Links
[Guides Project Readme](https://github.com/hackforla/guides/blob/main/README.md)
[Guides Team One-Sheet](https://github.com/hackforla/guides/issues/8)
[All Hack for LA Guides Shortcuts](https://drive.google.com/drive/folders/1buzmMJ5gCMvm3mpmdqDWryI6IBA8foKA)
[Organizations Google Drive: Hack for LA Guides](https://drive.google.com/drive/folders/1LAE_-KUd72u7vTBOTwU1Rz83rTCbeWB_?usp=sharing)
[Guides Team Figma](https://www.figma.com/file/Jz8KoGTBIxdx9jRxBWrEsF/Guides-Team-Figma?node-id=0%3A1)
[Communities of Practice](https://github.com/hackforla/communities-of-practice)
[All Guides Tracker Project board](https://github.com/orgs/hackforla/projects/3#card-72148101)
[Cleanup Guide Templates Issues](https://github.com/hackforla/guides/issues?q=is%3Aopen+is%3Aissue)
[Audit Spreadsheet for Tracking CoPs Guides](https://docs.google.com/spreadsheets/d/1mO3MPGPs3fD45XI1yuVeToqROS6n4rmY8ciGjVst9sc/edit#gid=0)
[Guides Team Slide Deck for Guide Creation](https://github.com/hackforla/guides/issues/44)
```
- [ ] Determine where this item goes on the wiki (page and section)
- [ ] copy item to wiki
- [ ] hide this comment when completed.

**Comment by edwardsarah** [comment]
### Text from project board card, Ice Box
```
Icebox = issues identified that are going to get addressed, but are not being started yet because of other more pressing issues.  Or issues that are not fully developed.
```
- [ ] ~Determine where this item goes on the wiki (page and section)~
- [ ] ~copy item to wiki~
- [ ] Revised text, added back to project board
- [ ] hide this comment when completed.



Issues that are going to get addressed, but have a blocker (usually another issue)

**Comment by edwardsarah** [comment]
### Text from project board card, New Issue Approval
```
Where new issues get started and become ready for moving into the prioritized backlog
```
- [ ] Determine where this item goes on the wiki (page and section)
- [ ] copy item to wiki
- [ ] hide this comment when completed.

**Comment by edwardsarah** [comment]
### Text from project board card, New Issue Approval
```
Add new team members to Google Drive, Have them add themselves to the roster, Add them to Figma
```
- [ ] Determine where this item goes on the wiki (page and section)
- [ ] copy item to wiki
- [ ] hide this comment when completed.

**Comment by edwardsarah** [comment]
### Text from project board card, New Issue Approval
```
Bi-Weekly check on Guides that need leadership review
[Product Management](https://github.com/hackforla/product-management/issues?q=is%3Aopen+is%3Aissue+label%3A%22TG%3A+Leadership+Review%22)
[UI/UX](https://github.com/hackforla/UI-UX/issues?q=is%3Aopen+is%3Aissue+label%3A%22Guide%3A+Leadership+Review%22)
```
- [ ] Determine where this item goes on the wiki (page and section)
- [ ] copy item to wiki
- [ ] hide this comment when completed.

**Comment by edwardsarah** [comment]
### Text from project board card, New Issue Approval
```
Follow up with this guide author
https://github.com/hackforla/knowledgebase-content/issues/84
```
- [ ] Determine where this item goes on the wiki (page and section)
- [ ] copy item to wiki
- [ ] hide this comment when completed.

**Comment by edwardsarah** [comment]
### Text from project board card, New Issue Approval
```
Make issue for emojis (they need to be useable across platforms) see emojiterra.com for compatability
```
- [ ] Determine where this item goes on the wiki (page and section)
- [ ] copy item to wiki
- [ ] hide this comment when completed.

**Comment by edwardsarah** [comment]
### Text from project board card, In Progress (Actively Working)
```
In Progress = tasks that are actively being worked on by an individual team member and have been assigned to them in the issue.
```
- [ ] Determine where this item goes on the wiki (page and section)
- [ ] copy item to wiki
- [ ] hide this comment when completed.

**Comment by edwardsarah** [comment]
### Text from project board card, Questions/Review
```
need to add new text to issues under peer review section
https://github.com/hackforla/knowledgebase-content/issues/127
- [x] Review the guide with UX/UI community of practice
   - [ ] Make any changes indicated by comments
   - [ ] Ask for new reviews until no suggestions for improvements are left
   - [ ] Once done, remove the "Guide: Review Guide" label and add the "Guide: Leadership Review" label
```
- [ ] Determine where this item goes on the wiki (page and section)
- [ ] copy item to wiki
- [ ] hide this comment when completed.

**Comment by edwardsarah** [comment]
### Text from project board card, Questions/Review
```
Questions/Review =  Issues that you are blocked on because you need an answer from another team member or other type of question/review.
```
- [ ] Determine where this item goes on the wiki (page and section)
- [ ] copy item to wiki
- [ ] hide this comment when completed.

**Comment by edwardsarah** [comment]
### Text from project board card, To Be Addressed
```
### Check the Wiki to update it ###
```
- [ ] Determine where this item goes on the wiki (page and section)
- [ ] copy item to wiki
- [ ] hide this comment when completed.

**Comment by edwardsarah** [comment]
### Text from project board card, To Be Addressed
```
### Aeding indicated on Slack that she would like to rejoin the team. https://hackforla.slack.com/archives/C028T9XU9S5/p1659704557772719 ###
```
- [ ] Determine where this item goes on the wiki (page and section)
- [ ] copy item to wiki
- [ ] hide this comment when completed.

**Comment by edwardsarah** [comment]
### Text from project board card, To Be Addressed
```
### We need to define some other Onboarding and Offboarding situations ###
```
- [ ] Determine where this item goes on the wiki (page and section)
- [ ] copy item to wiki
- [ ] hide this comment when completed.

**Comment by edwardsarah** [comment]
### Text from project board card, To Be Addressed
```
### We need to create an issue or a guide that explains all the labels in Hack For LA and how/when to use them ###
```
- [ ] Determine where this item goes on the wiki (page and section)
- [ ] copy item to wiki
- [ ] hide this comment when completed.

**Comment by edwardsarah** [comment]
### Text from project board card, To Be Addressed
```
### We need to add Guide for making guides issue/templates, draft, and the guide for adding screenshots to the Resource Section of new guide template across all CoPs. See Product Management Guides for reference ###
```
- [ ] Determine where this item goes on the wiki (page and section)
- [ ] copy item to wiki
- [ ] hide this comment when completed.

**Comment by ExperimentsInHonesty** [comment]
### Text from project board card, Prioritized Backlog
```
Prioritized Backlog = features that are ready to work and a priority.  This column should be prioritized by most important first.
```
- [ ] Determine where this item goes on the wiki (page and section)
- [ ] copy item to wiki
- [ ] hide this comment when completed.
